### ISSUE TITLE
Training on evaluator

### ISSUE THREAD

**Issue by tasbolat1** [jsll/pytorch_6dof-graspnet#15]
Hi @jsll,
Thanks for the wonderful work. I have been trying to use train.py with continue_train is being set to true. For this I set options same as pretrained_evaluator/opt.yml:

`python train.py --arch evaluator --continue_train True --dataset_root_folder /home/tasbolat/GRASP/unified_grasp_data/ --batch_size 350 --niter 1000 --niter_decay 10000 --num_grasps_per_object 70 --num_objects_per_batch 5 --num_threads 3`

However, it is not training as expected. The test accuracy directly jumps around 67% (from 78% initially) and loss goes up quickly. Can you advice which parameter i am passing wrong? Thanks
